packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -254,6 +258,8 @@ function Tab({
       tabIndex={isSelected ? 0 : -1}
       aria-controls={tabContentId}
       data-testid="workspace-tab-button"
+      data-connectionName={connectionName}
```

Comment:
I added these data attributes so we can be sure we're addressing the right tab.

`connectionName` because even with COMPASS-8038 I think it will be awkward to address. There's only title & subtitle and the breadcrumb is up to four levels deep for a view, not counting the fact that there can be different views for the same breadcrumb (Databases & Shell, for example.)

`type` because title & subtitle will always be ambiguous on their own because the title for the shell tab is the connection name and once we do COMPASS-8049 the Databases tab will also be named after the connection name. The visible difference is the icon.